plugins/catalog-backend-module-github/api-report.md

```diff
@@ -65,14 +65,36 @@ export class GitHubEntityProvider implements EntityProvider {
   refresh(logger: Logger): Promise<void>;
 }
 
+// @public
+export class GithubEntityProvider implements EntityProvider {
+  // (undocumented)
+  connect(connection: EntityProviderConnection): Promise<void>;
+  // (undocumented)
+  static fromConfig(
+    config: Config,
+    options: {
+      logger: Logger;
+      schedule?: TaskRunner;
+      scheduler?: PluginTaskScheduler;
+    },
+  ): GithubEntityProvider[];
+  // (undocumented)
+  getProviderName(): string;
+  // (undocumented)
+  refresh(logger: Logger): Promise<void>;
+}
+
 // @alpha
 export const githubEntityProviderCatalogModule: (
   options?: undefined,
 ) => BackendFeature;
 
+// @public @deprecated (undocumented)
+export class GitHubLocationAnalyzer extends GithubLocationAnalyzer {}
+
 // @public (undocumented)
-export class GitHubLocationAnalyzer implements ScmLocationAnalyzer {
-  constructor(options: GitHubLocationAnalyzerOptions);
```

Comment:
These were added in #13800 and haven't been part of a main line release yet, so it's fine to rename these immediately. If you want to you can update the existing changeset and changelog entry instead as well.

Reply:
sounds good. I will adjust it.